code/modules/mob/login.dm

```diff
@@ -94,3 +96,21 @@
 		hotkeyname = hotkeys[client.hotkeyon ? "on" : "off"]//get the name of the hotkey, to not clutter winset() to much
 		client.hotkeyon = 0
 	winset(src, null, "mainwindow.macro=[hotkeyname] hotkey_toggle.is-checked=false input.focus=true input.background-color=#D3B5B5")
+
+/client
+	var/obj/screen/spessbg = null
+
+/client/proc/create_parallax()
+	if (src.spessbg)

```

Comment:
okay i already got my line spamming done w ith
all of these should not have `src.`, it's always implied